axelor-account/src/main/java/com/axelor/apps/account/db/repo/AccountAccountManagementRepository.java

```diff
@@ -0,0 +1,35 @@
+package com.axelor.apps.account.db.repo;
+
+import com.axelor.apps.account.exception.IExceptionMessage;
+import com.axelor.exception.AxelorException;
+import com.axelor.exception.db.repo.TraceBackRepository;
+import com.axelor.i18n.I18n;
+import java.util.Map;
+import javax.persistence.PersistenceException;
+
+public class AccountAccountManagementRepository extends AccountManagementRepository {
+
+  @Override
+  public Map<String, Object> validate(Map<String, Object> json, Map<String, Object> context) {
+    boolean alreadyExists =
+        all()
+                .filter(
+                    "self.interbankCodeLine = :interbankCodeLine and self.bankDetails = :bankDetails and self.paymentMode = :paymentMode")
```

Comment:
we should check if the map contains "id", else this method could detect the account management we are trying to save